CONTRIBUTING.md

```diff
@@ -25,22 +25,22 @@ See the [SMQTK README] file for additional information.
 
         $ git push origin HEAD -u
 
-5.  Visit the Kitware SMQTK page [on GitHub], browse to the "Pull requests" tab
+5.  Visit the Kitware SMQTK-Core page [on GitHub], browse to the "Pull requests" tab
     and click on the "New pull request" button in the upper-right.
     Click on the "compare across forks" link, browse to your fork and browse to
     the topic branch to submit for the pull request.
     Finally, click the "Create pull request" button to create the request.
 
 
-SMQTK uses GitHub for code review and [Travis-CI] for continuous testing as new
+SMQTK-Core uses GitHub for code review and [Travis-CI] for continuous testing as new
```

Comment:
Change the mention of Travis-CI here to Github Actions. We can probably remove the link aspect of this (and the associated reference at the bottom here) since I think the location of github actions is obvious.